src/components/Progress.jsx

```diff
@@ -2,10 +2,18 @@ import React from 'react'
 import {formatCents} from '../tools'
 import {ProgressBar} from '.'
 
-export const Progress = ({data: {progress_percentage, donated_amount_in_cents, requested_amount_in_cents}}) => {
+const Collected = ({params}) => {
+  const collected = params.get('collected')
+  if (!collected || 0 === collected.length) return 'gesammelt.'
```

Comment:
```suggestion
  if (!collected || 0 === collected.length) return ' gesammelt.'
```

Sonst geht hier das Leerzeichen verloren, das unten stand?